src/api/firebase.js

```diff
@@ -193,10 +199,37 @@ export async function updateItem(listPath, item) {
 	const listCollectionRef = collection(db, listPath, 'items');
 	const itemRef = doc(listCollectionRef, item.id);
 
+	const currentDate = Timestamp.now();
```

Comment:
I would recommend to use built-in JavaScript `Date.now()` instead of Firebase's `Timestamp`

Reply:
So in this instance we chose to use Firebase's Timestamp because the other dates that we are dealing with are also in the Firebase Timestamp format. We could have either converted all the Timestamps to JavaScript dates and then converted those to milliseconds with math, OR (as we decided) we could have kept all the times as the Firebase Timestamp format and used the Firebase method `toMillis()` to easily convert them to milliseconds. This also allowed us to use the `toMillis()` method in the `getDaysBetweenDates` function and required less code than using the JavaScript date format.